sherpa/plot/tests/test_bokeh.py

```diff
@@ -0,0 +1,55 @@
+#
+#  Copyright (C) 2024
+#  MIT
+#
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License along
+#  with this program; if not, write to the Free Software Foundation, Inc.,
+#  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+"""This module collects tests specific to bokeh."""
+
+import os
+import pytest
+from sherpa import plot
+from sherpa.astro import ui
+from sherpa.utils.testing import requires_data, requires_fits
+
+
+@requires_fits
+@requires_data
+def test_bokeh_delchi(caplog, clean_ui):
+    """Test a specific multi-panel plot with the bokeh backend.
+
+    This also implictly test setting a line_color, because
+    delchi put a black horizontal line on the plot.
+
+    This is a regression test for https://github.com/sherpa/sherpa/issues/1975
+    """
+    path = os.path.join(
+        os.path.dirname(__file__),
+        "..",
+        "..",
+        "astro",
+        "datastack",
+        "tests",
+        "data",
+        "3c273.pi",
+    )
+    ui.load_pha(path)
+    ui.set_source(1, ui.powlaw1d(name="powlaw"))
+
+    bokehbackend = pytest.importorskip("sherpa.plot.bokeh_backend")
+    newback = bokehbackend.BokehBackend()
+    with plot.TemporaryPlottingBackend(newback):
```

Comment:
If we add any more tests we probably want to add a fixture to `conftest.py` - `requires_bokeh` to complement the new `requires_pylab` fixture.

Reply:
If and when we add more tests. One thing that's beyond the scope of this PR is to actually test the output. bokeh plots get turned into json (which is then passed to the browser), so we can do tests that check the text of plot labels like `assert "counts / s / keV" in outputplot_json`. Those things are hard to test properly in matplotlib! I envision that we add tests like that over time and then we'll see if we need a global or local fixture for this.

Reply:
I don't think any of our CI runs include bokeh do they? Or did we change that when you added #1899 - I have a feeling that you did that, but am too lazy to look myself

Reply:
https://github.com/sherpa/sherpa/commit/87090c67bd42fce4288cbeba9465b02bd5cb3573